metrics/util/util.go

```diff
@@ -21,29 +21,59 @@ import (
 	v1listers "k8s.io/client-go/listers/core/v1"
 	kube_api "k8s.io/client-go/pkg/api/v1"
 	"k8s.io/client-go/tools/cache"
+	"k8s.io/heapster/metrics/core"
 	"sort"
 	"strings"
 	"time"
 )
 
 var labelSeperator string
+var storedLabels map[string]string
+var ignoredLabels map[string]string
 
-// Concatenates a map of labels into a Seperator-seperated key:value pairs.
-func LabelsToString(labels map[string]string) string {
-	output := make([]string, 0, len(labels))
-	for key, value := range labels {
-		output = append(output, fmt.Sprintf("%s:%s", key, value))
+// Copies labels from input map to output:
```

Comment:
godoc should be of the form

`<method name> does a thing`, so here it should look like:

```
// CopyLabels copies the given set of pod labels into a set of metric labels, using the following logic:
// - all labels....
// ...
```